db/seeds.rb

```diff
@@ -6,3 +6,18 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+admin_email = ENV.fetch('ADMIN_USER', 'admin@peatio.tech')
+admin = Account.create(email: admin_email, password: 'P@ssword', role: 'admin', confirmed_at: Time.now)
+p "ADMIN email: #{admin.email} pass: #{admin.password}"
```

Comment:
Please generate a 16 chars random password and use rails log to print it.
Also it would be nice to ask to change it on first login if devise has this feature